services/arrow-flight/src/test/java/com/dremio/service/flight/TestFlightServerWithTokenAuth.java

```diff
@@ -43,10 +44,14 @@ protected String getAuthMode() {
   }
 
   @Override
-  public FlightInfo getFlightInfo(String query) {
+  CallOption[] getCallOptions() {
```

Comment:
There are lots of unrelated test changes - I think this is refactoring to get the testing portion ready to properly handle Flight SQL tests of the sort you're now doing?

Reply:
Sorry about this @kylep-dremio - yes, this is to properly handle both legacy and FlightSql and both basic and token authentication methods